README.md

```diff
@@ -185,14 +168,34 @@ Sample screenshot:
 
 ![](images/sensor.[zone_name].png?raw=true)
 
+#### Services
+
+| Service | Description|
+| --- | --- |
+|`Smart Irrigation: calculate_zone`|Triggers the calculation of one specific zone.|
+|`Smart Irrigation: calculate_all_zones`|Triggers the calculation of all automatic zones. Use only if you disabled automatic refresh in the options.|
+|`Smart Irrigation: update_zone`|Updates one specific zone with weather data|
+|`Smart Irrigation: update_all_zones`|Updates all automatic zones with weather data|
+|`Smart Irrigation: reset_bucket`|Resets one specific bucket.|
+|`Smart Irrigation: reset_all_buckets`|Resets all automatic buckets.|
+|`Smart Irrigation: set_bucket`|Sets a specific bucket to to a specific `value`.|
+|`Smart Irrigation: set_all_buckets`|Sets all automatic buckets to a specific `value`.|
+
+#### Start Event
+
+| Event | Description|
+| --- | --- |
+|`[instance]_start`|Fires depending on `sensor.[zone_name]` value and sunrise. You can listen to this event to optimize the moment you irrigate so your irrigation starts just before sunrise and is completed at sunrise. See below for examples on how to use this.|
```

Comment:
no, this event fires on the total of the durations of all non-disabled (so automatic or manual) zones, not just one. the math to determine when to fire this event is: sunrise - sum(duration for all non-disabled zones).

Reply:
aha... would be fine in my issue. ;-)
done